### PR TITLE
feat: add ChangeAvatarCard component

### PR DESCRIPTION
**Pull request by Mayowa-Ojo**
#2174

This PR:
- adds the `ChangeAvatarCard` component
- add the story component
- implements file change event. Logic will be added later

### CODE REVIEW

File: kibbeh/src/ui/ChangeAvatarCard.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+   return (
+      <BaseSettingsItem className="flex items-center px-4 py-3">
+         <div>
+            <SingleUser src={avatarUrl} />
+         </div>
+         <div className="flex flex-col ml-5">
+            <div className="flex items-center">
+               <Button size="small" color="secondary" className="">
+                  <label htmlFor="avatar" className="relative cursor-pointer">
+                     Change profile picture
```
Comment (ajmnz) [inline kibbeh/src/ui/ChangeAvatarCard.tsx:37]: This should be `text-primary-100`

**Review by ajmnz — CHANGES_REQUESTED** [review]

File: kibbeh/src/ui/ChangeAvatarCard.tsx (hunk 1 above)
Comment (Mayowa-Ojo) [reply kibbeh/src/ui/ChangeAvatarCard.tsx:37]: You mean on the label? If yes, the button already has that style

Comment (ajmnz) [reply kibbeh/src/ui/ChangeAvatarCard.tsx:37]: Yeah, it's white rn

Comment (Mayowa-Ojo) [reply kibbeh/src/ui/ChangeAvatarCard.tsx:37]: Ok. done

**Pull request merged**